subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc

```diff
@@ -100,8 +100,9 @@ Specifies the Java home for the Gradle build process. The value can be set to ei
 _A reasonable default is derived from your environment (`JAVA_HOME` or the path to `java`) if the setting is unspecified._
 
 `org.gradle.jvmargs=(JVM arguments)`::
-Specifies the JVM arguments used for the Gradle Daemon. The setting is particularly useful for <<#sec:configuring_jvm_memory,configuring JVM memory settings>> for build performance. This does not affect the JVM settings for the Gradle client VM.
-_The default is `-Xmx512m "-XX:MaxMetaspaceSize=256m"`._
+Specifies JVM arguments to be used for the Gradle Daemon. The setting is particularly useful for <<#sec:configuring_jvm_memory,configuring JVM memory settings>> for build performance. This does not affect the JVM settings for the Gradle client VM.
+Supplied values are merged with the defaults, with special handling for <<#sec:configuring_jvm_memory,JVM memory settings>> (see link).
+_The default values are `-Xms256m -Xmx512m -XX:MaxMetaspaceSize=256m -XX:+HeapDumpOnOutOfMemoryError`._
```

Comment:
```suggestion
Specifies the JVM arguments to use for the Gradle Daemon. The setting is particularly useful for <<#sec:configuring_jvm_memory,configuring JVM memory settings>> for build performance. This does not affect the JVM settings for the Gradle client VM.
Gradle merges supplied values with the defaults, with special handling for <<#sec:configuring_jvm_memory,JVM memory settings>>.
_The default value is `-Xms256m -Xmx512m -XX:MaxMetaspaceSize=256m -XX:+HeapDumpOnOutOfMemoryError`._
```
Suggestion: slight reword for docs consistency. I suspect we should specify the default *value* here (not values) because the default value refers to the singular `org.gradle.jvmargs` property, not the plural JVM arguments.